Functions/src/main/java/com/epam/engx/cleancode/functions/task2/SitePage.java

```diff
@@ -20,10 +16,23 @@ public SitePage(String siteGroup, String userGroup) {
     }
 
     public String getEditablePageUrl(Map<String, String> params) {
-        String paramsString = "";
+        return composeUrl(params);
+    }
+
+    private String composeUrl(Map<String, String> params) {
+        StringBuilder stringBuilder = new StringBuilder(URL_OF_EDITABLE);
+        addParameters(params, stringBuilder);
```

Comment:
I think better to not use parameters as a return value. There is already low level of abstraction because sb created and used. 
`stringBuilder.append(getParams(params)).append(getAttributes());`